utils/ansible-playbooks/deploy-monitoring/templates/ocp-monitor.j2

```diff
@@ -279,7 +279,7 @@ else
 fi
 # Check disk space of Registry
 echo -n "$(datestamp) Registry disk usage: "
-POD=$(oc -n default get pods -l deploymentconfig=docker-registry -o name | sed 's#pod/##' | head -n 1)
+POD=$(oc -n default get pods -l deploymentconfig=docker-registry -o=custom-columns=NAME:.metadata.name | head -n 1)
```

Comment:
Needs `--no-header` else you just get the Header line.

```
# oc -n default get pods -l deploymentconfig=docker-registry -o=custom-columns=NAME:.metadata.name | head -n 1
NAME
```

Reply:
Fixed these up.